Jenkinsfile

```diff
@@ -9,6 +9,9 @@ pipeline {
   libraries {
     lib('fxtest@1.10')
   }
+  triggers {
+    pollSCM('*/5 * * * *')
```

Comment:
Can we define both a poll SCM *and* cron? I think it makes sense for most project to run at least once a day.

Reply:
Oh, great point.  While I wasn't intending to do away with the in-job-config crons, if/since I'm specifying this, here, we might as well/should move that to here as well, for the reason you say.

Thanks, I'll take a look at doing so.

Reply:
Fixed, I think!